cmd/testrunner.go

```diff
@@ -71,6 +71,7 @@ func setupTestCommand() *cobraext.Command {
 	cmd.PersistentFlags().StringP(cobraext.ReportFormatFlagName, "", string(formats.ReportFormatHuman), cobraext.ReportFormatFlagDescription)
 	cmd.PersistentFlags().StringP(cobraext.ReportOutputFlagName, "", string(outputs.ReportOutputSTDOUT), cobraext.ReportOutputFlagDescription)
 	cmd.PersistentFlags().BoolP(cobraext.TestCoverageFlagName, "", false, cobraext.TestCoverageFlagDescription)
+	cmd.PersistentFlags().StringP(cobraext.TestCoverageFormatFlagName, "", "cobertura", cobraext.TestCoverageFormatFlagDescription)
```

Comment:
Do we need to add the flag in the buildkite pipelines?

Reply:
This will be added to integrations, not here.

Reply:
Exactly, this would be added in the integration.

In this PR (not ready at this moment), it has been added this new flag parameter:
https://github.com/elastic/integrations/pull/8787/files#diff-68c0021e761ebe6b8a42d32dfdf3dda2200f0f2fee50106c85c9aea530ccf3c3R19 

Probably, it would be good to update also here (elastic-package) the BK pipelines to generate the artifacts in the same coverage format as in the integrations repository. WDYT @jsoriano ?

Reply:
Ok, as you prefer.